server/dba/seeders/prod/04-forestSpecies.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
```

Comment:
I believe in sequelize, you can specify 1:many relationships, but I have not seen that. Is that something that we should specify?

Reply:
new story (# 68) created to handle this.